nexus/src/sagas.rs

```diff
@@ -648,6 +648,10 @@ async fn ensure_region_in_dataset(
         // NOTE: This'll require updating the crucible agent client.
         id: RegionId(region.id().to_string()),
         volume_id: region.disk_id().to_string(),
+        encrypted: region.encrypted(),
+        cert_pem: None,
+        key_pem: None,
+        root_pem: None,
```

Comment:
Can we add a tracking issue for "adding support for disk encryption" to Nexus, if we aren't going to add it now?

(deferring the implementation is fine, I just wanna make sure we don't lose track of these spots where we're passing null values)

Reply:
Created https://github.com/oxidecomputer/omicron/issues/691